mozilla/blurts-server
# Mntor 2182/add nimbus feature rollout

# References: 
Jira: MNTOR-2182




# Description

- Add Nimbus support for enabling Monitor feature flags
  - replace fxa/telemetry ID with Glean-supported `experimentationId`
- Pare down the feature flag admin UI
  - only expose add, delete, and enable
  - adding a flag completely overrides Nimbus, and removing puts Nimbus back in control

# How to test

See the bottom of `getExperiments.ts`,  it returns an example `features` result and logs the targeting info, if the server is run in dev mode.

# Checklist (Definition of Done)
